composable/assetlist.json

```diff
@@ -1,9 +1,9 @@
 {
     "$schema": "../assetlist.schema.json",
-    "chain_name": "composable",
+    "chain_name": "centauri",
```

Comment:
the chain_name must remain as 'composable' matching the folder name.

Reply:
ah, is the folder name updatable?

Reply:
for live chains, we avoid that since it could break dependencies already established to look at it.